src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -822,8 +826,9 @@ void GNAPlugin::LoadNetwork(const CNNNetwork& _network) {
         passes->registerPass<EltwiseSplitOverChannelsPass>();
         passes->registerPass<InsertSplitAligningFilterPass>();
 
-        passes->registerPass<InsertCopyLayerPass>();
-
+        if (!isNgraphPassesUsed) {
```

Comment:
why do we need this condition?

Reply:
@marcinkusm legacy pass is still used for some deprecated tests. When they will be removed, we can clean up code.

Reply:
I just wonder if we should remove lagacy code and tests if they are no longer needed

Reply:
@marcinkusm Actually it was a plan) We have a task for rewriting deprecated tests to ngraph based. Currenty this task is postponed due to luck of resources.